src/aura/CRLP_Rollup/CRLP_RollupHelper.js

```diff
@@ -160,9 +160,8 @@
             cmp.set("v.detailFields", newFields);
         }
 
-        //todo: is this necessary? pulled in my commit
         var currentMode = cmp.get("v.mode");
-        if (currentMode !== 'view') {
+        if (currentMode === 'create') {
```

Comment:
This resets the field to null for a new record, but does it prevent the field from being reset to null when cloning?

Reply:
Yep! 'clone' is its own separate mode, so this fixes that problem too.